rllib/models/torch/torch_modelv2.py

```diff
@@ -41,39 +41,6 @@ def __init__(self, *args, **kwargs):
             name,
             framework="torch")
 
-    def forward(self, input_dict, state, seq_lens):
```

Comment:
Duplicate code in a child class.

Reply:
I had this before for clarity. Do you think it makes it less clear how to implement the model if it's removed?

Reply:
Fair point. I'll move it back. Removing the entire method in TorchModel breaks the linter anyways due to some abstract meta-class lint error (I think. it has to do with the fact that TorchModel is an nn.Module as well).

Reply:
Sounds good. Should also do it for the TF model too then.